### DISCUSSION TITLE
Images and Web Links

### DISCUSSION THREAD

**Discussion by ghost** [AzuraCast/AzuraCast#5449] [Ideas]
1.) Image associated with the Album Title instead of song
2.) Connect song/album/artist to a point of sale (custom link to POS database)

I have 0ver 65,000 songs and connecting an image to each song will take quit a long time which is why an image should be associated with the Album Title.
I have a POS system and connecting the song through the player would help sales. This is one of the features that is available in the old radio control panel I had been using for the past 10 years and one of the reasons I didn't make the change until May 20, 2022.

**Comment by cpaschen** [comment]
I think I have probably been using the 'old radio control panel' that you used.
Having a purchase link (or just for any purpose) would be very helpful at the song level.

**Reply by ghost** [reply]
I was using CentovaCast if you have to know, I didn't want to use the name here because it's not polite and that control panel I was using was recently update. The issue I had with them is they are a closed source program and the volume was inside the closed source and the volume is set to 100% on.

**Comment by BusterNeece** [comment]
This should probably be a feature request over at https://features.azuracast.com/

**Reply by ghost** [reply]
trying to login at AzuraCast with githup credentials I get an error

Oops, something went wrong...
You are not authorized to login.

I don't find a register link for AzuraCast would you post the feature request for me?

**Reply by Vaalyn** [reply]
You don't need to register an account or be logged in to one to post feature requests on that page.

**Reply by ghost** [reply]
not true, if it were I wouldn't be posting here telling you about the error. You need to check the login process for the Azuacast web site. I can not log-in or register at AzuraCast and I hate using git which is confusing and annoying

**Reply by Vaalyn** [reply]
Here is a video showcasing how to post a new feature request on https://features.azuracast.com/ without any registration / login:

https://user-images.githubusercontent.com/13745863/172249280-eff9e7f0-33f3-422b-9d90-4d1fd47e1350.mov

Posting a feature request over at that page has nothing to do with git.

**Reply by ghost** [reply]
thanks for the video but that doesn't fix the issue that I can not create an account with Azuracast web site or log-in to AzuaraCast web site using github credentials

**Reply by Vaalyn** [reply]
The https://features.azuracast.com/ that we use for feature requests does not have a login page.

The main https://azuracast.com website also does not have logins or registrations.

The only AzuraCast website that has a login page where one could potentially enter GitHub credentials is our Wiki (the docs website) but that has nothing to do with feature requests nor the main AzuraCast website and that login page is not for users. The login on the Wiki / Docs page is only for moderators and administrators at this point in time.

**Reply by ghost** [reply]
I guess I must be wrong
![azuracast-login](https://user-images.githubusercontent.com/50125233/172298151-2fd80feb-0b83-4f9d-b876-88f214c70ade.jpg)

**Reply by BusterNeece** [reply]
@440music You are!

I don't think you've been reading the replies here. There is no reason for you to log into our Wiki, and that's not how you submit feature requests. You submit them through https://features.azuracast.com/, a completely separate system that does not, as we've repeatedly explained, require a login.

**Reply by ghost** [reply]
I did post to the in the feature request and I've read every message
